ledger/metrics/src/main/scala/com/daml/metrics/api/noop/Metrics.scala

```diff
@@ -22,5 +22,9 @@ sealed case class NoOpTimer(name: String) extends Timer {
   ): T = call
```

Comment:
Nit: There's not much else in this file, i.e., we may want to rename it to simply `NoOpTimer`.